### PR TITLE
Use variable to call functions in devmapper_wrapper + some formatting

### PR DESCRIPTION
**Pull request by creack**

### CODE REVIEW

File: graphdriver/driver.go [hunk 1]
```diff
@@ -64,14 +63,8 @@ func GetDriver(name, home string) (Driver, error) {
 	return nil, fmt.Errorf("No such driver: %s", name)
 }
 
-func New(root string) (Driver, error) {
-	var driver Driver
-	var lastError error
```
Comment (crosbymichael) [inline graphdriver/driver.go:69]: You should not have removed this variable, it's now undefined

File: graphdriver/driver.go [hunk 2]
```diff
@@ -64,14 +63,8 @@ func GetDriver(name, home string) (Driver, error) {
...
-
-	for _, name := range []string{
```
Comment (crosbymichael) [inline graphdriver/driver.go:71]: Why did you put this on one line ?

File: graphdriver/driver.go (hunk 1 above)
Comment (creack) [inline graphdriver/driver.go:69]: fixed

File: graphdriver/driver.go (hunk 2 above)
Comment (creack) [inline graphdriver/driver.go:71]: Why not? I like oneliner :) No need to have multiple lines here.

**Comment by creack** [comment]
rebased. /cc @crosbymichael @vieux

**Comment by crosbymichael** [comment]
@creack 

I'm getting build errors on this branch.

```
# github.com/dotcloud/docker/graphdriver/devmapper
graphdriver/devmapper/devmapper.go:189: undefined: os
graphdriver/devmapper/devmapper.go:197: undefined: os
graphdriver/devmapper/devmapper_wrapper.go:243: undefined: syscall
graphdriver/devmapper/devmapper_wrapper.go:250: undefined: syscall
graphdriver/devmapper/devmapper_wrapper.go:251: undefined: syscall
```

**Comment by creack** [comment]
rebase fail.. Updated and retested.

**Comment by crosbymichael** [comment]
LGTM

ping @vieux

**Comment by vieux** [comment]
LGTM

**Pull request merged**